plugins/workbench-resources/src/components/Workbench.svelte

```diff
@@ -170,6 +168,22 @@
 
   let isNavigate: boolean = false
   $: isNavigate = !!navigatorModel
+
+  function navigateApp (app: Application) {
+    currentApp = app._id
+    currentApplication = app
+    navigatorModel = currentApplication?.navigatorModel
+
+    currentSpace = undefined
```

Comment:
may be we should compare currentApp with new and clear only if change?

Reply:
Thanks, resolved.